server/tournaments/__init__.py

```diff
@@ -0,0 +1,11 @@
+"""
+The team matchmaking system
+
+Used to keep track of current player parties, manage players joining/leaving and matching them against each other
+used for matchmaking in the global rating system
+"""
+__all__ = [
+    "TournamentGame"
+]
```

Comment:
I just recently learned that PEP 8 actually says to put the `__all__` variable before the imports. That probably wants to be changed all over the place in this code because I've been doing it the other way around. I've also been using tuples instead of lists. Not sure if that's a good thing though :P